### PR TITLE
Dumb hfstol

### PR DESCRIPTION
**Pull request by eddieantonio**
Because the `hfstol` library has issues, I've started to implement a dumb version that simply calls `hfst-optimized-lookup` for every time it's called. This is inefficient but ¯\\\_(ツ)\_/¯ .

@Madoshakalaka, I've written the first part of this. Once this is merged, could you please use this in `models.py`?

### CODE REVIEW

File: CreeDictionary/CreeDictionary/hfstol.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""
+Temporary HFSTOL.
```
Comment (eddieantonio) [inline CreeDictionary/CreeDictionary/hfstol.py]: This code is uglyyyyy 🙈🙈🙈

Comment (Madoshakalaka) [reply CreeDictionary/CreeDictionary/hfstol.py]: whatever works is beautiful

**Comment by Madoshakalaka** [comment]
@eddieantonio Do you want to re run the tests after #327 is merged?

**Comment by eddieantonio** [comment]
@Madoshakalaka: yup: rerunning now!

File: CreeDictionary/CreeDictionary/hfstol.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+@contextmanager
+def write_file(text: str) -> Generator[IO[str], None, None]:
+    with TemporaryFile("w+", encoding="UTF-8") as tmp:
+        tmp.write(text)
+        tmp.flush()
+        tmp.seek(0)
+        yield tmp
+
+
+def analyze(wordform: str) -> Generator[Analysis, None, None]:
```
Comment (Madoshakalaka) [inline CreeDictionary/CreeDictionary/hfstol.py]: aside from descriptive analyzer, normative generator is also used in the [search code] (https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/cf6057755f949501c5dc728fb2e1be55431f1c3e/CreeDictionary/API/models.py#L382-L403) to generate normative forms when the user gives analyzable words outside of the paradigm.

so we need a `generate` function. Do you think we need a separate file, a separate function in this same file or do you prefer other methods?

Comment (eddieantonio) [reply CreeDictionary/CreeDictionary/hfstol.py]: I'll write a separate `generate()` function in this file!

**Pull request merged**